camel-sap/pom.xml

```diff
@@ -18,73 +24,367 @@
 		<version>2.17.0.redhat-630133</version>
 	</parent>
 
-	<groupId>org.fusesource</groupId>
+	<groupId>org.fusesource.components</groupId>
 	<artifactId>camel-sap-parent</artifactId>
 	<version>6.3.0.redhat-SNAPSHOT</version>
 	<packaging>pom</packaging>
 
-	<name>JBoss Fuse :: Components :: SAP JCO</name>
+	<name>JBoss Fuse :: Components :: SAP</name>
 	<description>Provides SAP Camel Component</description>
 	<url>http://www.jboss.org/products/fuse/overview/</url>
 
+	<organization>
+		<name>Red Hat</name>
+		<url>http://www.redhat.com</url>
+	</organization>
+
+	<issueManagement>
+		<system>jira</system>
+		<url>https://issues.jboss.org/browse/ENTESB</url>
+	</issueManagement>
+
+	<scm>
+		<connection>scm:git:git://git@github.com:jboss-fuse/fuse-components.git</connection>
+		<developerConnection>scm:git:ssh://git@github.com:jboss-fuse/fuse-components.git</developerConnection>
+		<url>https://github.com/jboss-fuse/fuse-components</url>
+	</scm>
+
+	<prerequisites>
+		<maven>3.6</maven>
+	</prerequisites>
+
 	<properties>
-		<tycho-version>0.24.0</tycho-version>
-		<!-- JBoss Tools Integration Stack version -->
-	    	<jbtis.version>4.4.3.Final</jbtis.version>
-		
-		<maven.compiler.target>1.7</maven.compiler.target>
-		<maven.compiler.source>1.7</maven.compiler.source>
-		<maven-surefire-plugin-version>2.19</maven-surefire-plugin-version>
+		<project.reporting.outputEncoding>UTF-8</project.reporting.outputEncoding>
+		<project.build.sourceEncoding>UTF-8</project.build.sourceEncoding>
+
+		<maven.compiler.target>1.8</maven.compiler.target>
```

Comment:
i think that in 6.3 stream we need to stick with 1.7 as it is a supported java version https://access.redhat.com/articles/310603#Red-Hat-JBoss-Fuse-63